conversations.yaml

```diff
@@ -0,0 +1,536 @@
+                format: int64
+              message:
+                properties:
+                  mid:
+                    type: string
+                  text:
+                    type: string
+                    description:
+                      The message text, no longer than 1,600 characters.
+                      The channel service may split the message into
+                      multiple parts for delivery.
+              trigger_parameters:
+                properties:
+                  override_session:
+                    type: boolean
+                  consumer:
+                    properties:
+                      id:
+                        type: string
+                      email:
+                        type: string
+                  custom:
+                    description:
+                      A dictionary with the custom parameters, mapped
+                      to fields in Medallia Reporting via survey
+                      parameters. Those are some example values
+                    properties:
+                      session_id:
+                        type: string
+                      channel:
+                        type: string
+                      survey_type:
+                        type: string
+                      queue:
+                        type: string
+                      e_unitid:
+                        type: string
+    DeliveryStatusUpdate:
+      properties:
+        id:
+          type: string
+          description:
+            The PageId value configured in Conversations at the creation of the channel.
+        time:
+          type: integer
+          format: int64
+          description:
+            Unix epoch timestamp of the last event associated with
+            the request. For example, when it is a multi-part message,
+            this is the time of the last message-part.
+        messaging:
+          type: array
+          items:
+            properties:
+              sender:
+                properties:
+                  id:
+                    type: string
+                    description:
+                      An identifier unique to the customer, such as
+                      phone number, email, or other channel-specific
+                      value. For phone numbers, the ID must be in
+                      E.164 format.
+              recipient:
+                properties:
+                  id:
+                    type: string
+                    description:
+                      The PageId value configured in Conversations at the creation of the channel.
+              timestamp:
+                type: integer
+                format: int64
+              delivery:
+                properties:
+                  mids:
+                    type: array
+                    items:
+                      type: string
+                    description:
+                      (required) Identifier for the messages sent as
```

Comment:
Remove `(required)`.  Use the `required:` mechanism in OAS/Swagger to define what is/isn't required.